Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean

```diff
@@ -205,13 +205,12 @@ theorem Basis.parallelepiped_reindex (b : Basis ι ℝ E) (e : ι ≃ ι') :
 
 theorem Basis.parallelepiped_map (b : Basis ι ℝ E) (e : E ≃ₗ[ℝ] F) :
     (b.map e).parallelepiped = b.parallelepiped.map e
-    (have := FiniteDimensional.of_fintype_basis b
+    (haveI := FiniteDimensional.of_fintype_basis b
```

Comment:
Was there a specific reason for this change? (On my machine, it compiles fine without.)

I'm not sure if this is needed. I vaguely remember it is not; [this zulip thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/.60HaveI.60in.20statements.2Fproofs) seems to be somewhat in favour.